Products/ZenUI3/browser/resources/js/zenoss/zenoss.js

```diff
@@ -1237,19 +1237,13 @@ Ext.define("Zenoss.DateRange", {
             return errors;
         }
         //Look first for invalid characters, fail fast
-        if (/[^0-9TO :-]/.test(value)) {
+        if (/[^0-9TOampm :-]/.test(value)) {
             errors.push("Date contains invalid characters - valid characters include digits, dashes, colons, and spaces");
             return errors;
         }
         if (value.indexOf("TO") === -1) {
-            if (!Zenoss.date.regex.ISO8601Long.test(value)) {
-                errors.push("Date is formatted incorrectly - format should be " + Zenoss.date.ISO8601Long);
-                return errors;
-            }
-        }
-        else {
-            if (!Zenoss.date.regex.ISO8601LongRange.test(value)) {
-                errors.push("Date range is formatted incorrectly - format should be " + Zenoss.date.ISO8601LongRange.replace(/\\/g, ''));
+            if (!moment(moment(value, Zenoss.date.TimeFormats).format('YYYY-MM-DD HH:mm:ss').isValid())
+                errors.push("Date is formatted incorrectly - format should be " + Zenoss.USER_DATE_FORMAT + ' ' + Zenoss.USER_TIME_FORMAT);
```

Comment:
I think this error message will advise a format not supported by the code.  On my system, the error message would be
```
Date is formatted incorrectly - format should be YY/MM/DD hh:mm:ss a
```